mavsdk_server/src/main/java/io/mavsdk/mavsdkserver/MavsdkServer.java

```diff
@@ -63,10 +63,38 @@ public int run(String systemAddress, int mavsdkServerPort) {
     return getPort(mavsdkServerHandle);
   }
 
+  /**
+   * Run MavsdkServer with MAVLink on `systemAddress` with custom MAVLink IDs.
+   *
+   * <p>MavsdkServer will listen for a `System` to connect on `mavsdkServerPort`.</p>
+   *
+   * @param systemAddress The address on which the remote MAVLink system is expected.
+   *     Valid formats are:
+   *     For TCP : tcp://[server_host][:server_port]
+   *     For UDP : udp://[bind_host][:bind_port]
+   *     For Serial : serial:///path/to/serial/dev[:baudrate]
+   * @param mavsdkServerPort The port on which the server should listen for a `System`.
+   * @param systemId The identifier which the system should connect with.
```

Comment:
Maybe something more like this?

```suggestion
   * @param systemId The MAVLink sysid that MAVSDK should use.
```